stable_baselines/deepq/experiments/train_mountaincar.py

```diff
@@ -24,13 +15,14 @@ def main(args):
 
     # using layer norm policy here is important for parameter space noise!
     model = DQN(
-        policy=CustomPolicy,
+        policy="LnMlpPolicy",
         env=env,
         learning_rate=1e-3,
         buffer_size=50000,
         exploration_fraction=0.1,
         exploration_final_eps=0.1,
-        param_noise=True
+        param_noise=True,
+        policy_kwargs=dict(layers=[64])
```

Comment:
shouldn't you add `layer_norm` to the policy kwargs?

Reply:
if you do so, you get a `multiple keyword argument` error because `layer_norm=False` in the MlpPolicy, that's why I'm using the `LnMlpPolicy` as base policy.